resources/lib/criterion_parser.py

```diff
@@ -20,7 +20,7 @@ def parse_criterion(criterion):
 
     value = criterion.get('value', '')
     if isinstance(value, dict) and 'depth' in value:
-        if 'items' in value:
+        if 'items' in value and value['items'] is not None:
```

Comment:
I believe this could be:
```suggestion
        if value.get('items') is not None:
```
`.get()` will return `None` if the key doesn't exist (and in the filter I tested with an exclusion `items` wasn't present, that's why I changed it, as `value['items']` would otherwise result in an error). The same should then be applied to `'excluded' in value` as well as I guess that can exist as `None` too then.

Reply:
That makes sense, good change